# ITS_PCR_METHOD
Anna A. Hippmann  
January, 2016  

##Rational

I use two strains of the open ocean diatom _Thalassiosira oceanica_. Nemely, close CCMP 1003 and CCMP 1005. Only fo rht elatter strains, there is th egenome published. I have an EST library for the first strain. Growing them both under various Fe and Cu concentrations showed, that they have very different Fe and Cu needs in their media. To tease out how similar / different they are on the genomic level, I will sequence their ITS region / 5.8S rRNA region (as suggested by Bethany Jenkins)

## General Background on use of different genomic regions to determine phylogeny (barcoding organisms)

There are regions in the genome that evolve faster or slower. Depending on the rate of evolution, these regions are more or less suitable to determine differences between strains, families, or genus of two organisms(White et al, 1990).

- __slow evolving region__ e.g. the nuclear small subunit rRNA (16S-like), would be suitable to study distantly related organisms

- __intermediately evolving region__ e.g. mitochondrial rRNA genes, more suitable to discern ordinal/family level

- __fast evolving region__ e.g. internal transcribed spacer region and intergenic spacer of the nuclear rRNA repeat units, may vary among species within a genus or even within a population

__

##Primers Used

Here are the generic ITS primers that I use (covers the entire ITS1, 5.8S, and ITS2 region):

__ITS1: TCCGTAGGTGAACCTGCGG (forward)__
__ITS4: TCCTCCGCTTATTGATATGC (reverse)__

These are from White et al, 1990 . 

The PCR protocol I use is from Amato et al, 2007: 

- 2 min at 94C 
  + 35s at 94C 
  + 35s at 46.2
  + 60s at 72C 
  +(paper says 40 cycles but as sugested by studetn of Bethany, I will start with just 35 cycles)
- 5 min at 72C

If those don't work for whatever reason then Moniz and Kaczmarska, 2010 have suggested using ITS5 as the forward primer or the pair NS7m/LR1850.

## Preparing the Primers

as per Flo:

I got the dry primers

- ITS_4R -> + 250 ul H2O
- ITS_1F -> + 209 ul H2O

this will result in __100 uMol stock solutions__ of each primer
use __10uM stock soltn  for PCR__ ( 0.5-1 uM could be sufficient, too)

(Ao uses __0.25uL__ of each stocksoltn __in 50uL PCR reaction mix__ -> 0.5uM)

## Preparing the cultures

How much DNA do I need? 

How mcuh culture do I need to grow for this?

Let's assume one 27ml culture tube would be enough!

I would inoculate 2x 27ml culture tubes per strain (TO 1003, TO 1005, TP (for good measure))

- spin down in big centrifuge
- in decant SN
- combine pellets into one tube, refill / resuspend / wash with sterile Aquil
- spin down, transfer pellet into eppendorf tube
- spin down
liq N2

-> then either DNA rausholen
-> or to Michael Murphy

